zfit_physics/models/pdf_tsallis.py

```diff
@@ -147,12 +151,14 @@ def __init__(
             tf.debugging.assert_greater(n, znp.asarray(2.0), message="n > 2 is required")
 
         params = {"m": m, "t": t, "n": n}
-        super().__init__(obs=obs, params=params, name=name, extended=extended, norm=norm)
-
-    def _unnormalized_pdf(self, x: tf.Tensor) -> tf.Tensor:
-        m = self.params["m"]
-        t = self.params["t"]
-        n = self.params["n"]
+        super().__init__(obs=obs, params=params, name=name, extended=extended, norm=norm, label=label)
+
+    @zfit.supports()
+    def _unnormalized_pdf(self, x: tf.Tensor, params) -> tf.Tensor:
+        m = params["m"]
+        t = params["t"]
+        n = params["n"]
+        x = x[0]
         return tsallis_pdf_func(x=x, m=m, t=t, n=n)
```

Comment:
Just FYI @ikrommyd , that's what 0.20.0 has in it: the params are now given as an argument (therefore, the method could be static!) and the x can now be accessed via the "axis" (i.e. the x is a Data object, but indexing with 0 returns the 0th axis)

The whole PR is just a chore, an upgrade of internals

Reply:
but I see this change only in zfit_physics and not in zfit for now

Reply:
Partially, the zfit pdfs have all labels by now (which is needed in the plotting) and some also get the params. It really doesn't change a lot, it's more about the API and how it looks for the user, and we're able to make them static. `zfit.supports()` provides the magic :)